src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -33,12 +39,29 @@
     TokenClassifierOutput,
 )
 from ...modeling_utils import PreTrainedModel
-from ...utils import logging
+from ...utils import is_ninja_available, logging
 from .configuration_bloom import BloomConfig
+from .load_custom import load_cuda_kernels
+from .parallel_layers import TensorParallelColumnLinear, TensorParallelEmbedding, TensorParallelRowLinear
 
 
 logger = logging.get_logger(__name__)
 
+
+fused_bloom_attention_cuda = None
+
+
+def use_custom_kernels():
```

Comment:
Why define a function? Not a particular fan of `global`, `deformable_detr` seems to have done so without defining such function.

Reply:
So are you saying that one would do something along the lines of:
```
from transformers import BloomModel
model = BloomModel.from_pretrained("bigscience/bloom")

from transformers.models.bloom.modeling_bloom import use_custom_kernels
use_custom_kernels()
model.generate(...)
```
This seems weird to me. I guess we could make it a context somehow or things like that? But again. I feels like we don't need custom kernel yet (it doesn't support backward, it's never been tested on something else than A100) and we get most of the speed up from the TP implementation.